File: app/src/main/java/com/dmi/perfectreader/render/RenderChild.java
package com.dmi.perfectreader.render;

public class RenderChild {
    public final float x;
    public final float y;
    public final RenderObject object;

    public RenderChild(float x, float y, RenderObject object) {
        this.x = x;
        this.y = y;
        this.object = object;
    }

    public float x() {
        return x;
    }

    public float y() {
        return y;
    }

    public RenderObject object() {
        return object;
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/render/RenderChild.kt
package com.dmi.perfectreader.render

class RenderChild(val x: Float, val y: Float, val `object`: RenderObject) {

    fun x(): Float {
        return x
    }

    fun y(): Float {
        return y
    }

    fun `object`(): RenderObject {
        return `object`
    }
}
